README.md

```diff
@@ -561,6 +562,15 @@ To delete a local branch:
 
 ## Rebasing and Merging
 
+<a name="undo-rebase">
+### I want to undo rebase/merge
+
+You may have merge/rebase your current branch with a wrong branch, or you simply cannot manage to achieve a correct result or even finish the rebase/merge process. Git is saving the original HEAD pointer in ORIG_HEAD, so it is simple to recover your branch at the state before the rebase/merge.
```

Comment:
@gaetan-petit Do you want to add this to the PR and edit it? Or do you want me to?